dataAccessObjects.py

```diff
@@ -283,6 +296,9 @@ def getImgSystemName(self,imgId):
         sql = 'SELECT img_system_name FROM image_data WHERE img_id="{imgId}"'.format(imgId=imgId)
         return self.queryOneValue(sql)
 
+    def updateImgEditedData(self,data):
```

Comment:
You can delete this function, because the derived class can use the method of parent class